tests/conftest.py

```diff
@@ -96,3 +99,18 @@ def example_model_with_logs(example_model_with_well):
    log_collection.add_log("NPHI", [0.1, 0.1, np.NaN, np.NaN], 'v/v')
 
    return model, well_interp, datum, traj, frame, log_collection
+
+
+@pytest.fixture
+def example_data_path(tmp_path):
+   """ Return pathlib.Path pointing to temporary copy of tests/example_data
+
+   Use a fresh temporary directory for each test.
+   """
+   master_path = (Path(__file__) / '../example_data').resolve()
```

Comment:
We currently have two directories called example_data: one at the top level for example RESQML datasets for people to play with, and one within the test directory.  I suggest we rename the one in the test directory as test_data. (Doesn't need to be part of this change but I thought I'd mention it here before forgetting about it again!)

Reply:
@andy-beer  @connortann  - I did notice there were two, though the example_data folder within the test directory contains a couple of extra files. I'm not sure if/where these files are used - does anyone know where these are used so I can update the references to these at the same time? If not I might break something when renaming

Reply:
I think you're fine to go ahead and rename the tests/example_data directory. I think I added that LAS file for a test of `resqpy.well.add_las_to_trajectory`, but haven't yet got round to writing that test!